### ISSUE TITLE
The download feature is not working properly anymore

### ISSUE THREAD

**Issue by eyeslow** [iKarwan/YTKillerPlus#20]
Hey there, since the latest release **18.16.2 (2.2.9)** the download feature is not working as it should. 
Instead of showing the resolutions or audio format it just says **(null)**.

I'm **not jailbroken** and installed it via **AltStore on 16.4.1** and via **Sideloadly on 15.7.1**. 
Both devices are unfortunately showing the same bug, which was not present in previous versions.

Thank You! :)

**Comment by eyeslow** [comment]
Hello again, the temporary fix with version **2.2.9-1** unfortunately doesn't fix the situation. It still does not list any resolution for video downloads and even crashes the application when I'm trying to download the audio stream.

Take your time and don't worry, we will survive a few days or even weeks without the download feature!😝

**Comment by eyeslow** [comment]
Now, I‘m able to happily inform you, that the (second) temporary fix with version **2.2.11** has fixed the issues!🥳 
Only thing is, that the download speed is very slow, but I‘m sure you‘ll get down to this as well, as this version was explicitly a _temporary fix._ Thank you very much for the work you‘re putting into your tweaks!

Have a good one :)

**Comment by eyeslow** [comment]
> hey there 
> 
> 
> 
> pls remove the ipa on appdb if u can i will be appreciated to u really

Hey there, I‘m at school right now and will remove the (temporarily) cracked version in my break soon. Should I also remove the legit, not cracked, variant? I‘ve just uploaded this version so I could still watch my videos and at least access the app and my downloaded videos.
I've also got another question for you, I've purchased more than one license but not all are active at the moment, so they're not linked to a UUID at the moment, how would I be able to keep them active?
